### PR TITLE
feat: add more comments in all lu editors

### LINKED ISSUE #957: LU default copy should show sample
Currently, LU by default says : >To learn more about the LU file format, read the documentation at > https://github.com/Microsoft/botbuilder-tools/blob/master/packages/Ludown/docs/lu-file-format.md#lu-file-format

Issue: 

- the link is not clickable
- the github page has a bunch of info, all I need to know is what to type to get started.
- I already gave the trigger a name (getweather), this should be shown 

Much more useful would be :
#Getweather  
- < add a sentence or word that the user can say to trigger 'Getweather' >
- < add an alternative way to say the same sentence or word to trigger 'Getweather' >

### PR DESCRIPTION
**Pull request by liweitian**
## Description
add comments to give users some suggestions

## Task Item
Closes #957 
## Screenshots
微信图片_20200424172437
微信图片_20200424172448
微信图片_20200424172452

### CODE REVIEW

File: Composer/packages/lib/code-editor/src/LuEditor.tsx [hunk 1]
```diff
@@ -13,7 +13,13 @@ import { BaseEditor, BaseEditorProps, OnInit } from './BaseEditor';
...
 const LU_HELP = 'https://aka.ms/lu-file-format';
 const placeholder = `> To learn more about the LU file format, read the documentation at
-> ${LU_HELP}`;
+> ${LU_HELP}
+> add some example phrases to trigger this intent:
+> - please tell me the weather
+> - what is the weather like in {city=Seattle}
+> entity definitions: 
+> @ ml city
+`;
```
[the remark this answers is not in the document]
Comment (liweitian) [reply Composer/packages/lib/code-editor/src/LuEditor.tsx:22]: @cwhitten 
微信图片_20200421152640
do we need to add these comments in all up view?

[the remark this answers is not in the document]
Comment (cwhitten) [reply Composer/packages/lib/code-editor/src/LuEditor.tsx:22]: Good point @a-b-r-o-w-n. Let's break it apart to say the following:

In all-up LU page:

> \> To learn more about the LU file format, read the documentation at
> \> https://aka.ms/lu-file-format

In the in-line editors, _remove_ the above section, so we would see the following:

> \> add some example phrases to trigger this intent:
> \> - please tell me the weather
> \> - what is the weather like in {city=Seattle}
> \> entity definitions: 
> \> @ ml city

[the remark this answers is not in the document]
Comment (liweitian) [reply Composer/packages/lib/code-editor/src/LuEditor.tsx]: done

**Comment by liweitian** [comment]
> I'm with Andy on this. Passing a key around through props is unusual; I think if you're going to pass a string around, it should just be the placeholder text itself without the extra layer of indirection. If you don't want to repeat the text, there should be some kind of consistent place text constants like that go. They will also need to be wrapped in formatMessage for translation purposes.

Done. placeholder msg is stored in @bfc/shared

**Pull request merged**